chrome/content/zotero/itemTree.jsx

```diff
@@ -2613,9 +2700,9 @@ var ItemTree = class ItemTree extends LibraryTree {
 		}
 		let textSpanAriaLabel = [textWithFullStop, itemTypeAriaLabel, tagAriaLabel, retractedAriaLabel].join(' ');
 
-		let textSpan = document.createElementNS("http://www.w3.org/1999/xhtml", 'span');
+		let textSpan = document.createElementNS(HTML_NS, 'span');
 		textSpan.className = "cell-text";
-		textSpan.innerText = data;
+		this._renderItemTitle(data, textSpan);
```

Comment:
I think we'll want to use the resulting text content for the title part of the ARIA label above as well. Can we either return that (or use `textContent` on the resulting node, but that's probably slower) and use that instead of `data` to initialize `textWithFullStop`?